source/nodejs/adaptivecards/src/card-elements.ts

```diff
@@ -2008,6 +2008,17 @@ export abstract class CardElementContainer extends CardElement {
 
         return result;
     }
+
+    /**
+     * @inheritdoc
+     */
+    findObject(node: Node): CardObject | undefined {
+        let target;
+        for(let i = 0; i < this.getItemCount(); i++) {
+            target = target ?? this.getItemAt(i).findObject(node); // recur through child elements
+        }
```

Comment:
Won't this override target even when one is found? We should stop the loop immediately if a target was found.